packages/react-instantsearch/examples/next-app/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "create-next-example-app",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "build": "NODE_ENV=development next build",
```

Comment:
Well this is not working since some modifications with the .babelrc file to handle code splitting. Maybe @Haroenv could have a look.